pkg/bindata/templates/profile.html

```diff
@@ -18,31 +18,143 @@ <h1><i class="fas fa-user-alt" style="float:left;font-size:40px"></i> User Profi
+                                            </td>
+                                        </tr>
+                                        <tr>
+                                            <td>
+                                                <div style="display:none;" class="dialogText">
+                                                    If Notifiarr client is behind a reverse proxy (swag, nginx, etc), then you should add that proxy IP here. You can also specify full networks.
+                                                    This controls which IP is written to the web server log file (requestor or <span class="text-warning">x-forwarded-for</span> header).
+                                                    It also allows the <span class="text-warning">x-webauth-user</span> header for proxy authentication. Separate with spaces or new lines.<br>
+                                                    <b>Information</b>:<br>
+                                                    <i>Upstream</i>: <b>{{.UpstreamIP}}<</b><br>
+                                                    <i>X-Forwarded-For</i>: <b>{{.Headers.Get "x-forwarded-for"}}<</b><br>
+                                                    <b>Current Values</b>:<br>
+                                                    {{- range $i, $s := .Config.Allow.Input}}<i>{{instance $i}}</i>: <b>{{$s}}</b><br>{{end}}
+                                                    {{- if not .Config.Allow.Input}}<i>нет значения, ноль</i>{{end}}{{/* "no value, null" */}}
+                                                </div>
+                                                <a onClick="dialog($(this), 'left')" class="help-icon far fa-question-circle"></a>
+                                                <span class="dialogTitle">Upstreams</span>
+                                            </td>
+                                            <td class="mobile-hide">
+                                                {{range $i, $s := .Config.Allow.Input}}{{instance $i}}: <b>{{$s}}</b><br>{{end}}{{if not .Config.Allow.Input}}<i>no value, null</i>{{end}}
+                                            </td>
+                                            <td class="form-inline">
+                                                <div class="form-group" style="width:100%">
+                                                    {{- if (locked (printf "%s_UPTREAMS_0" .Flags.EnvPrefix))}}
+                                                    <div class="input-group" style="width:100%">
+                                                        <div style="width:30px; max-width:30px;" class="input-group-addon input-sm">
+                                                            <div style="display:none;" class="dialogText">
+                                                                An environment variable exists for this value. Your new value will write to the config file, but the application will not use it.
```

Comment:
```suggestion
                                                                An environment variable exists for this value. Your new value will write to the config file, but the application will not use the config file value.
```

Reply:
If you want this changed, open a PR and change every instance.